dev/run_gitlab_in_docker.sh

```diff
@@ -1,130 +1,149 @@
-#!/bin/bash
-
-set -euo pipefail
-
-# based on https://stackoverflow.com/a/28709668/2693875 and https://stackoverflow.com/a/23006365/2693875
+#!/usr/bin/env bash
```

Comment:
I love that you refactored this script, thank you so much @mahadevan-karthi-dwp! ☺️ 🤝

Reply:
I could not squash these commits, will be opening a new PR with same changes. Just incase if you wonder why I close this one.